src/main/java/com/google/sps/servlets/GetProductDetails.java

```diff
@@ -59,6 +61,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
     long id = Long.parseLong(request.getParameter("id"));
 
     long cost;
+    double rating = 0;
```

Comment:
I think double by default have value 0.0, thus you can remove the "=0".

Reply:
It should, but the compiler through an uninitialized variable warning, so I put it there to be safe.